website/docs/registry/modules/verified.mdx

```diff
@@ -1,25 +1,25 @@
 ---
-page_title: Terraform Registry - Verified Modules
+page_title: Terraform Registry - Partner Modules
 description: >-
-  Verified modules are reviewed by HashiCorp and actively maintained by
-  contributors to stay up to date.
+  Partner modules are actively maintained by partner
+  contributors.
 ---
 
-# Verified Modules
+# Partner Modules
 
-Verified modules are reviewed by HashiCorp and actively maintained by contributors to stay up-to-date and compatible with both Terraform and their respective providers.
+Partner modules are published by tech and cloud organizations that have a partnership agreement with HashiCorp. Partner contributors actively maintain their modules to stay up-to-date and compatible with both Terraform and their respective providers.
 
-The verified badge appears next to modules that are published by a verified source.
+The partner badge appears next to modules that are published by a partner.
 
 ![Verified module listing](/img/docs/registry-verified.png)
 
-Verified modules are expected to be actively maintained by HashiCorp partners.
-The verified badge isn’t indicative of flexibility or feature support; very
-simple modules can be verified just because they're great examples of modules.
-Likewise, an unverified module could be extremely high quality and actively
-maintained. An unverified module shouldn't be assumed to be poor quality, it
+Partner modules are expected to be actively maintained by HashiCorp partners.
```

Comment:
Rewrite in active voice and suggested we remove the "expected to be" because it feels vague:
```suggestion
HashiCorp partners actively maintain partner modules.
```